### ISSUE TITLE
Tensorflow Colab - Custom training: walkthrough

### ISSUE THREAD

**Issue by giovanni-turra** [googlecolab/colabtools#383]
Good Morning,

I found an error inside step 20 of "Custom training: walkthrough" Tensorflow Colab.

Specifically, in the chapter training loop, it is necessary to fix:

`epoch_loss_avg = tf.metrics.Mean()`
`epoch_accuracy = tf.metrics.Accuracy()`

with:

`epoch_loss_avg = tf.contrib.eager.metrics.Mean()`
`epoch_accuracy = tf.contrib.eager.metrics.Accuracy()`

And similar in the chapter "Evaluate the model on the test dataset".

Thank you!

**Comment by colaboratory-team** [comment]
Thanks for the report. We'll pass this along to the TensorFlow documentation team.
(b/122104457)
